webots_ros2_driver/README.md

```diff
@@ -1 +1,13 @@
 # Implementation of the interface for Webots and ROS 2
+
+The main functionalities of the package: 
+- It automatically creates a ROS 2 interface out of a Webots robot model.
+- It allows users to configure the ROS 2 interface in URDF files.
+- It allows users to extend the interface using the pluginlib plugin mechanism.
+
+> This package aims to replace the `webots_ros2_core` package.
+
+Please use the following links for more details:
+
+* [Tutorial: Creating Custom Package](https://github.com/cyberbotics/webots_ros2/wiki/Tutorial-Creating-Custom-Package)
+* [Tutorial: Creating Custom Plugin](https://github.com/cyberbotics/webots_ros2/wiki/Tutorial-Creating-Custom-Plugin)
```

Comment:
I didn't check the details of these wiki page, but the title sounds odd, I would change it to:
```suggestion
* [Tutorial: Creating a Custom Package](https://github.com/cyberbotics/webots_ros2/wiki/Tutorial-Creating-a-Custom-Package)
* [Tutorial: Creating a Custom Plugin](https://github.com/cyberbotics/webots_ros2/wiki/Tutorial-Creating-a-Custom-Plugin)
```